Add powerview update entity information (#23057)
home-assistant/home-assistant.github.io: source/_integrations/hunterdouglas_powerview.markdown

- Scene
- Sensor

<div class="note">

> Shade positions in Home Assistant represent only what the Powerview Hub reports. Unfortunately positional changes made with pebble remotes do not sync to the hub.
> Forcing an update for all shades would result in excessive battery drain so this is not done on a schedule. Users who wish to implement this can call the service `homeassistant.update_entity`. [An example automation is avaiable](#force-update-shade-position)

Reviewer: ```suggestion
The Powerview Hub does not automatically wake shades or report position changes made via pebble remotes.

Calling the update entity service (`homeassistant.update_entity`) on a shade entity will trigger the hub to awaken a shade and report its current position. [An example automation is avaiable](#force-update-shade-position) below for mains powered shades. While the automation will work for battery-powered shades, it will quickly drain their batteries for these devices.
```

Reviewer: You might need to tweak that a bit more, but I think it is important to make it clear they shouldn't use the automation with battery powered devices since they may not make the connection that the automation will drain the battery.

- Scene
- Sensor

<div class="note">
The Powerview Hub does not automatically wake shades or report position changes made via pebble remotes.

> Calling the update entity service (`homeassistant.update_entity`) on a shade entity will trigger the hub to awaken a shade and report its current position.  [An example automation is avaiable](#force-update-shade-position) below for mains powered shades.  While the automation will work for battery-powered shades, it will quickly drain their batteries for these devices.

Reviewer: ```suggestion
Calling the update entity service (`homeassistant.update_entity`) on a shade entity will trigger the hub to awaken a shade and report its current position.  [An example automation is available](#force-update-shade-position) below for mains powered shades.  While the automation will work for battery-powered shades, it will quickly drain their batteries for these devices.
```

> Calling the update entity service (`homeassistant.update_entity`) on a shade entity will trigger the hub to awaken a shade and report its current position.  [An example automation is available](#force-update-shade-position) below for mains powered shades.  While the automation will work for battery-powered shades, it will quickly drain their batteries for these devices.

Reviewer: ```suggestion
Calling the update entity service (`homeassistant.update_entity`) on a shade entity will trigger the hub to awaken a shade and report its current position. [An example automation is available](#force-update-shade-position) below for mains powered shades. While the automation will work for battery-powered shades, it will quickly drain their batteries for these devices.
```

## Example Automations

### Calling a Powerview Scene

``` yaml
alias: "blinds closed at night"
trigger:
  platform: time
  at: "18:00:00"
action:
  - service: scene.turn_on
    target:
      entity_id: scene.10877
```
### Force Update Shade Position

> ``` yaml

Reviewer: ```suggestion
This automation is not recommended for battery-powered shades.

``` yaml
```

## Example Automations

### Calling a Powerview Scene

``` yaml
alias: "blinds closed at night"
trigger:
  platform: time
  at: "18:00:00"
action:
  - service: scene.turn_on
    target:
      entity_id: scene.10877
```
### Force Update Shade Position

This automation is not recommended for battery-powered shades.

``` yaml
alias: Force Update
description: 'Update the position of defined shades'
mode: single
trigger:
  - platform: time_pattern
    hours: '1'

> condition: []

Reviewer: ```suggestion
```

Reviewer: I'm not sure if thats needed but need to check

action:
  - service: homeassistant.update_entity

>     data: {}

Reviewer: ```suggestion
```

99% sure this isn't needed

Author: likely not - was just a straight copy out of an automation i created in thr i
